_posts/2020-09-28-Kamen-Kanev-cv.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: cv
+title: Kamen Kanev
+comment: Default CV
+---
+# Kamen Kanev
+Software Engineer and Entrepreneur
+
+<div id="webaddress">
+<a target="_blank" rel="noopener" href="tel:+359879666995">+359-879-666-995</a>
+| <a target="_blank" rel="noopener" href="mailto:kamen.e.kanev@gmail.com">kamen.e.kanev@gmail.com</a>
+| <a target="_blank" rel="noopener" href="https://www.linkedin.com/in/kamen-kanev-97889b116/">LinkedIn</a>
+| <a target="_blank" rel="noopener" href="https://github.com/kanevk">Github</a>
+</div>
+
+## Emphasis
+
+Curious and open-minded individual, passionate about forming mental models for complex problems and
+developing solutions that have a social impact.
+Experienced in designing APIs, working on data-heavy applications, building processes, and managing people.
+
+### Technologies
+
+Ruby on Rails, React, GraphQL, PostgreSQL, Python, Django, NodeJS, RSpec, AWS, Capistrano
+
+## Professional Experience
+
+### Founder / CEO
+
+`April 2020 - September 2020`
+__[Bakalin BG](https://bakalin.bg/){:target="_blank" rel="noopener"}__
+
+We at Bakalin BG have the ambition of making the local specialty stores competitive to the supermarket chains.
+I and my partners have built the service from scratch, and I've learned a lot in various areas:
+Product vision, UX, Leadership, Logistics, Legal, Negotiation.
+One of the most noticeable achievements of mine is innovating with a delivery strategy
+that has improved our profit predictions with up to 20% on a yearly base.
```

Comment:
a `year` we talked about this

Reply:
```
One of the most noticeable achievements of mine is contributing to a delivery dynamic pricing
that improved our business plan profit predictions with around 20%.
```

Reply:
Remove of `mine` and rephrase this: `delivery dynamic pricing`, as I do not understand what this means.